### DISCUSSION TITLE
Glow Berry Cookies

### DISCUSSION THREAD

**Discussion by noawx** [vectorwing/FarmersDelight#788] [Ideas]
Since in vanilla minecraft, glow berries have the same stats and functionality as sweet berries, there should be glow berry cookies to contrast sweet berry cookies.

**Comment by vectorwing** [comment]
Glow Berry Custard is the main dessert made from Glow Berries.

Main reason for that is because... I *tried* spriting Glow Berry Cookies before, and the colors did not match well at all. It looked unappetizing and blended together. ;_;

**Reply by vectorwing** [reply]
It's kinda hard to integrate their palette into most sprites, due to it looking like a glowing blob moreso than a berry. Always up for new attempts, though; just didn't pan out when I tried it.
